src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,33 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+// import todos from '../../api/todos';
+import { Todo } from '../../types/Todo';
```

Comment:
<img width="537" alt="Screenshot 2023-06-22 at 17 27 51" src="https://github.com/mate-academy/react_add-todo-form/assets/53228259/b67b830a-0c1a-447c-a2bb-a63b125eff16">
you forgot to push folder types